### DISCUSSION TITLE
Bridge mastodon hashtags to Bluesky feeds?

### DISCUSSION THREAD

**Discussion by stephenturner** [snarfed/bridgy-fed#1344] [Q&A] [answered]
I mostly use Bluesky but the #rstats hashtag on Mastodon is something I read through periodically: https://mastodon.social/tags/rstats

Is it possible to bridge all posts to #rstats on mastodon to a bluesky feed?

**Comment by snarfed** [comment, marked as answer]
Good question! Sorry, no, I have no plans to do that right now. Afaik ActivityPub doesn't have a way to "subscribe" to a tag feed like that, relays notwithstanding. Also it would be limited to a single instance, and it would only show opted in users, which would be a very small subset.
